### DISCUSSION TITLE
As We Speak: Real-Time Visually Guided Speaker Separation and Localization

### DISCUSSION THREAD

**Discussion by SutirthaChakraborty** [speechbrain/speechbrain#2709] [General]
Any plan to implement this kind of paper? For realtime ?  https://ieeexplore.ieee.org/document/9949329

**Comment by SutirthaBoss** [comment]
Any update?

**Reply by Adel-Moumen** [reply]
Hey, 

https://github.com/speechbrain/speechbrain/discussions/2709#discussioncomment-12549995

:)

**Comment by Adel-Moumen** [comment]
> Any plan to implement this kind of paper? For realtime ? https://ieeexplore.ieee.org/document/9949329

Hello, no we don't plan to add this paper in SpeechBrain. We are open to open-source contributions, but we don't have the bandwidth to implement this paper.
